docs/installation.md

```diff
@@ -0,0 +1,37 @@
+# Installation
```

Comment:
Can we add a Windows section? :)

Reply:
I'll see if I can find a Windows machine to test this on -- do we know what percentage of our audience is using Windows?

Reply:
I have said windows machine... but I'm in meetings from now until 4pm today. XD
We can always add it later.